tox.ini

```diff
@@ -62,6 +62,9 @@ commands =
 passenv =
     CACHI2_GENERATE_TEST_DATA
     CACHI2_IMAGE
+    CACHI2_TEST_LOCAL_PYPISERVER
+setenv =
+    CACHI2_TEST_NETRC_CONTENT={env:CACHI2_TEST_NETRC_CONTENT:machine localhost login cachi2-user password cachi2-pass}
```

Comment:
Why do we want to make `CACHI2_TEST_NETRC_CONTENT` variable configurable from the outside for test purposes? Isn't a fixed value enough? When would we want to reconfigure the test differently?

Reply:
I used it for example to see what happens when the password is wrong. It's not strictly necessary, but I would expect the tests to respect the env var if I try to override it